app/models/review.rb

```diff
@@ -0,0 +1,11 @@
+class Review < ApplicationRecord
+  validates :rating, presence: true
+  validates :rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }
+  validates :body, presence: true
+
+  belongs_to :band
+  belongs_to :user
+
+  validates :user_id, presence: true
+  validates :band_id, presence: true
```

Comment:
lines 9 and 10 are not needed in rails, as they are done automatically by lines 6 and 7